src/assets/TokenListController.ts

```diff
@@ -195,11 +219,20 @@ export class TokenListController extends BaseController<
    * Fetching token list from the contract-metadata as a fallback
    */
   async fetchFromStaticTokenList(): Promise<void> {
-    const tokenList: TokenMap = {};
-    for (const tokenAddress in contractmap) {
-      const { erc20, logo, ...token } = contractmap[tokenAddress];
+    const tokenList: TokenListMap = {};
+    for (const tokenAddress in contractMap) {
+      const { erc20, logo: filePath, ...token } = (contractMap as ContractMap)[
+        tokenAddress
+      ];
       if (erc20) {
-        tokenList[tokenAddress] = { ...token, iconUrl: logo };
+        // Specify iconUrl here as filePath for backwards compatibility for extension
```

Comment:
Nit: this comment was meant to explain why both `iconUrl` and `iconPath` were set, but now that `iconPath` is gone it is no longer necessary.